styx-cli/src/main/java/com/spotify/styx/cli/CliMain.java

```diff
@@ -784,4 +791,38 @@ public boolean consumeArgument() {
       return true;
     }
   }
+
+  interface CliContext {
+
+    CliOutput output(OUTPUT output);
+
+    enum OUTPUT {
```

Comment:
shouldn't the type name be cased like regular classes/types and only the values UPPERCASE?

Reply:
yeah this code was written very quickly 😂